sym/provider/flow_resource.go

```diff
@@ -55,65 +91,16 @@ func flowSchema() map[string]*schema.Schema {
 		"vars":           utils.SettingsMap("A map of variables and their values to pass to `impl.py`. Useful for making IDs generated dynamically by Terraform available to your `impl.py`. "),
 		"environment_id": utils.Required(schema.TypeString, "The ID of the Environment this Flow is associated with."),
 		"params": {
-			Type:             schema.TypeMap,
-			Required:         true,
-			DiffSuppressFunc: utils.SuppressFlowDiffs,
-			ValidateDiagFunc: validateParams,
-			Description:      "A set of parameters which configure the Flow. See the [Sym Documentation](https://docs.symops.com/docs/flow-parameters).",
+			Description: "A set of parameters which configure the Flow.",
+			Type:        schema.TypeList,
+			Optional:    true,
+			Computed:    true,
+			MaxItems:    1,
+			Elem:        flowParamsSchema(),
 		},
 	}
 }
 
-// Template Helper Functions ////////////////////
-
-// Remove the version from our template type for handling
-// e.g. sym:template:approval:1.0 becomes just sym:template:approval
-func getTemplateNameWithoutVersion(templateName string) string {
```

Comment:
Thought: Should we get rid of the `template` attribute in `sym_flow`, because it's no longer used/needed? There's only ever one template; previously we were using this key to figure out how to validate `params`. But now that `params` is a typed block that is always the same, we don't need it anymore.

Reply:
Discussing at Sync today https://coda.io/d/Cycle-Planning-Process_dFR1ciuFR8W/Cycle-95-doin-the-that-thing_suUCY#_lulax